tests/functionalComponents.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { mount } from '../src'
+import { h } from 'vue'
+import Hello from './components/Hello.vue'
+
+describe('functionalComponents', () => {
+  it('mounts a functional component', () => {
+    const Foo = (props) => h('div', { class: 'foo' }, props.msg)
+
+    const wrapper = mount(Foo, {
+      props: {
+        // @ts-ignore
+        msg: 'foo'
+      }
+    })
```

Comment:
```suggestion
    const Foo = (props: { msg: string }) =>
      h('div', { class: 'foo' }, props.msg)

    const wrapper = mount(Foo, {
      props: {
        msg: 'foo'
      }
    })
```

you can described the props on the function parameter :)